reactive/filebeat.py

```diff
@@ -76,6 +76,10 @@ def render_filebeat_template():
             )
         log(msg)
 
+    # run the sanity checks, abort the rendering if config is not valid
+    if not is_config_valid():
+        return
```

Comment:
I cannot verify it now because the build is failing on master branch, but here it's return without removing the `beat.render` state. This might cause a continues loop on trying to run this function

Reply:
ah great find!

Reply:
+1 to this one.

Reply:
great find, fixed, thanks